react/SeekApp/SeekApp.js

```diff
@@ -2,23 +2,46 @@ import styles from './SeekApp.less';
 
 import React, { PropTypes } from 'react';
 import classnames from 'classnames';
+import Helmet from 'react-helmet';
+import ScreenReaderOnly from 'Accessibility/ScreenReaderOnly';
 
-export default function SeekApp({ fullScreen, children }) {
+const defaultPageTitleAU = 'SEEK - Australia\'s no. 1 jobs, employment, career and recruitment site';
+const defaultPageTitleNZ = 'Jobs on SEEK - New Zealand\'s no. 1 Employment, Career and Recruitment site';
+
+const getLocalisedPageTitle = locale => locale === 'AU' ? defaultPageTitleAU : defaultPageTitleNZ;
+
+export default function SeekApp({ fullScreen, children, meta = [], link = [], title, locale = 'AU' }) {
   const className = classnames({
     [styles.root]: true,
     [styles.fullScreen]: fullScreen
   });
 
+  const pageTitle = title || getLocalisedPageTitle(locale);
+
   return (
     <div className={className}>
+      <Helmet
+        title={pageTitle}
+        meta={meta}
+        link={link}
+      />
+
+      <ScreenReaderOnly>
```

Comment:
If we hoist this up to the style guide is should probably be top level in the `react` directory and exported as part of the style guide package. Consuming apps should then refactor away their internal usage of `<ScreenReaderOnly />` in favour of this. We'll cover this in the release notes